README.md

```diff
@@ -149,6 +149,24 @@ For example, the following would use `index.ts` as the secondary entry point:
 }
 ```
 
+##### What if I want to use React Components?
+
+If you have React Components that you're using in your library, and want to use proper JSX/TSX syntax in order to
+construct them, you can set the `jsx` flag for your library through `ng-package` like so:
+
+```json
+{
+  "ngPackage": {
+    "lib": {
+      "entryFile": "public_api.ts",
+      "jsx": "react"
+    }
+  }
+}
+```
+
+Note: You probably also want to include `react` and `react-dom` in your `externals`.
```

Comment:
Is there a case where `react` or `react-dom` wouldn't be needed in `externals`? If not, it would probably stop a few support issues by including them in the example json.

Reply:
I'm having a hard time thinking of one. Perhaps you really want to include react in your bundle? But that's against what is suggested by `ng-packagr` I believe. I'll go ahead and add it to the example json.